pkg/classifier/nodes.go

```diff
@@ -0,0 +1,28 @@
+package classifier
+
+import "fmt"
+
+const (
+	uri = "/classifier-api/v1/classified/nodes/"
+)
+
+// Node will return the Node matching the given id.
+// certname is the hostname of the node to query.
+func (c *Client) Node(certname string) (Node, error) {
+	payload := Node{}
+	err := getRequest(c, fmt.Sprintf("%s/%s", uri, certname), nil, &payload)
+	return payload, err
+}
+
+// Node represents the response to the classifier nodes endpoint.
+type Node struct {
+	Name        string   `json:"name"`
+	Environment string   `json:"environment"`
+	Groups      []string `json:"groups"`
+	Classes     struct {
```

Comment:
I think the empty struct doesn't help here because the marshaller will always parse whatever is there into this empty struct giving nothing. If you omit it completely the JSON parser will ignore it and you get the same result. Or you can type it as `map[string]interface{}` so that arbitrary structure can be present. I'd probably go for the map.

Reply:
I am not sure if this is still the case @scottyw  ?

Reply:
Groups      []struct {
		ID   string `json:"id"`
		Name string `json:"name"`

Group's is now modeled inside the node. So we should be good to go.

Reply:
Hey, yes, looks good to me too.